Sources/StreamChat/Utils/TranslationLanguage.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct TranslationLanguage: Hashable {
+    public static let afrikaans: TranslationLanguage = TranslationLanguage(languageCode: "af")
+    public static let albanian: TranslationLanguage = TranslationLanguage(languageCode: "sq")
+    public static let amharic: TranslationLanguage = TranslationLanguage(languageCode: "am")
```

Comment:
why don't we get translation languages from the backend? 😅

```suggestion
 public struct TranslationLanguage: Hashable { }
    
 public extension TranslationLanguage {
    static let afrikaans = Self(languageCode: "af")
    ...
}
```

Reply:
¯\_(ツ)_/¯